game-core/src/test/java/org/triplea/client/ui/javafx/TripleAApplicationTest.java

```diff
@@ -0,0 +1,44 @@
+package org.triplea.client.ui.javafx;
+
+import static org.testfx.api.FxAssert.verifyThat;
+import static org.testfx.matcher.base.NodeMatchers.hasChildren;
+import static org.testfx.matcher.base.NodeMatchers.isInvisible;
+import static org.testfx.matcher.base.NodeMatchers.isVisible;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.sonatype.goodies.prefs.memory.MemoryPreferences;
+import org.testfx.api.FxToolkit;
+import org.testfx.framework.junit5.ApplicationExtension;
+
+import games.strategy.triplea.settings.ClientSetting;
+
+@ExtendWith(ApplicationExtension.class)
+public class TripleAApplicationTest {
+
+  static {
+    System.setProperty("java.awt.headless", String.valueOf(true));
+    System.setProperty("testfx.robot", "glass");
+    System.setProperty("testfx.headless", String.valueOf(true));
+    System.setProperty("prism.order", "sw");
+    System.setProperty("prism.text", "t2k");
+    System.setProperty("testfx.setup.timeout", String.valueOf(10000));
```

Comment:
Does the existing system use system properties for these values? 
I'd like for us to take a very close look at this, system props go badly wrong. Notice for example existing TripleA code that uses them to send messages from one part of the code to another.

I personally would lean to a configuration object, set that explicitly somewhere. Let's please discuss how this is used and if we like the system property pattern here.

Reply:
These system properties are just set for the lifetime of the JVM.
They are necessary to allow for headless builds, the alternative would be to modify the gradle script to set those properties when launching the tests, but I simply didn't wanted to bother with that for now...